cmd/ipfs/daemon.go

```diff
@@ -676,6 +676,10 @@ func serveHTTPApi(req *cmds.Request, cctx *oldcmds.Context) (<-chan error, error
 		listeners = append(listeners, apiLis)
 	}
 
+	if len(cfg.API.Authorizations) > 0 && len(listeners) > 0 {
+		fmt.Printf("RPC API access is limited by the rules defined in API.Authorizations\n")
```

Comment:
ℹ️ @hacdias fyi I've pushed a commit with this – just to be sure users who experiment with this new feature get notified when they start `ipfs daemon` with  it enabled.